packages/react-tinacms-github/src/github-client/index.ts

```diff
@@ -73,7 +72,18 @@ export class GithubClient {
     })
   }
 
-  async fetchExistingPR(forkRepoFullName: string, headBranch: string) {
+  get repoFullName(): string {
+    return getForkName() || this.baseRepoFullName
+  }
+
+  get branchName() {
+    return getHeadBranch()
+  }
+
+  async fetchExistingPR() {
+    const forkRepoFullName = this.repoFullName
```

Comment:
misleading `forkRepoFullName` variable if it falls back to  `getForkName() || this.baseRepoFullName`